index.bs

```diff
@@ -2031,7 +2065,9 @@ The <dfn export for=commands>browser.createUserContext</dfn> command creates a
 
 The [=remote end steps=] are:
 
-1. Let |user context| be a new [=empty user context=].
+1. Let |user context| be a new [=user context=].
```

Comment:
Should we keep the empty here to make sure there are no top-level navigables in the freshly created context?

Reply:
I don't think it's necessary. Whether or not a user context is empty is a computed property of the context, so talking about creating one as if that's directly controllable doesn't make sense. Moreover, at the moment a user context is created it can't have any navigables in it because all existing navigables must belong to a different user context, and nothing has created one in this user context, so I think it has to be empty.